scripts/dockerdev.sh

```diff
@@ -38,13 +35,15 @@ dockerdev () {
     fi
 
     # If already running, enter the container.
-    if $RUNTIME ps | grep -q $container_name; then
+    if $RUNTIME ps | grep -q $image_name:$CONTAINER_VERSION; then
```

Comment:
this seems wrong  `docker ps` shows the running containers, which are named with the container name, not the image name. If I have the same image running under a different name, I don't want `dockerdev` to use that.

Reply:
Good point, added a second filter :+1: